src/components/staking/LiValidator.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <tr
+    v-if="delegation.amount >= SMALLEST"
```

Comment:
please move this logic into the API as discussed

Reply:
into `resolvers` > `delegations`

Reply:
Yep, I'm working on it